src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1935,12 +1935,31 @@ namespace UnitarySystems {
                 SizingMethod = DataHVACGlobals::AutoCalculateSizing;
                 this->m_MaxNoCoolHeatAirVolFlow = DataSizing::AutoSize;
             } else if (this->m_NoCoolHeatSAFMethod == FlowPerCoolingCapacity) {
+                if ( EqSizing.DesCoolingLoad <= 0.0) {
```

Comment:
water coil cooling capacity not sized yet. If not sized, size the water coil capacity here.